src/symmbasis.jl

```diff
@@ -49,20 +49,17 @@ function SymmetricBasis(pibasis, φ::TP) where {TP}
    # TODO: should this be stored with the basis?
    #       or maybe written to a file on disk? and then flushed every time
    #       we finish with a basis construction???
-   rotc = Rot3DCoeffs(rfltype(pibasis))
-
+   #rotc = Rot3DCoeffs(rfltype(pibasis))
+   rotc = Rot3DCoeffsEquiv(φ, rfltype(pibasis))
```

Comment:
I take it your coefficients will take care of either invariant or equivariant properties? But will your code clash with Liwei's? We need to ensure this works with both.

Reply:
yes, that is correct! Rot3DCoeffsEquiv allows to compute coefficients both for invariant and covariant properties, and may be generalized to other symmetries. I did not want to overwrite Rot3DCoeffs, but also did not want to add redundant code (even though I think in the end I did). So that's why I introduced the abstract type R3DC

Reply:
let's move this to an issue